drivers/sensor/bm160/bm160.h

```diff
@@ -401,18 +499,30 @@ union bmi160_pmu_status {
 	};
 };
 
+#if defined(CONFIG_BMX160_MAG)
+#	define BMX160_MAG_SAMPLE_SIZE		(4 * sizeof(u16_t))
+#else
+#	define BMX160_MAG_SAMPLE_SIZE		(0)
+#endif
+
 #if !defined(CONFIG_BMI160_GYRO_PMU_SUSPEND) && \
 		!defined(CONFIG_BMI160_ACCEL_PMU_SUSPEND)
 #	define BMI160_SAMPLE_SIZE		(6 * sizeof(u16_t))
 #else
 #	define BMI160_SAMPLE_SIZE		(3 * sizeof(u16_t))
 #endif
 
-#define BMI160_BUF_SIZE			(BMI160_SAMPLE_SIZE)
+#define BMI160_BUF_SIZE		(BMI160_SAMPLE_SIZE + BMX160_MAG_SAMPLE_SIZE)
```

Comment:
Why `I` size has relation with magnetometer?

Reply:
Changes all the magnetometer related things to BMX but existing code will remains same. The idea is to make minimal changes to existing code to support for BMX magnetometer, consider change of common naming will leads to touch almost 70% of code.

Please share you feedback